CSharp/Library/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj

```diff
@@ -57,6 +57,7 @@
     <Reference Include="System.Core" />
     <Reference Include="System.Net" />
     <Reference Include="System.Net.Http.WebRequest" />
+    <Reference Include="System.Runtime.Caching" />
```

Comment:
>System.Runtime.Caching [](start = 24, length = 22)

this is available in .Net standard?

Reply:
Yes, [https://github.com/aspnet/Caching/tree/dev/src/Microsoft.Extensions.Caching.Memory](https://github.com/aspnet/Caching/tree/dev/src/Microsoft.Extensions.Caching.Memory) 

---
In reply to: [175547434](https://github.com/Microsoft/BotBuilder/pull/4332#discussion_r175547434) [](ancestors = 175547434)